cluster/gce/util.sh

```diff
@@ -1670,11 +1875,15 @@ function validate-node-local-ssds-ext(){
 # Robustly try to create an instance template.
 # $1: The name of the instance template.
 # $2: The scopes flag.
-# $3: String of comma-separated metadata entries (must all be from a file).
+# $3: String of comma-separated metadata-from-file entries.
+# $4: String of comma-separated metadata (key=value) entries.
+# $5: "linux" or "windows".
 function create-node-template() {
   detect-project
   detect-subnetworks
   local template_name="$1"
+  local metadata_values="$4"
+  local linux_or_windows="$5"
```

Comment:
we called this `os` elsewhere (I think I like this name better)

again though we should error out if something other than "linux" or "windows" makes its way here

Reply:
Done.